draft-todo-yourname-protocol.md

```diff
@@ -35,6 +42,9 @@ venue:
   arch: https://example.com/WG
   github: USER/REPO
   latest: https://example.com/LATEST
+keyword:
+ - TODO1
+ - TODO2
```

Comment:
I already have the most useful "keyword" possible above.  Changing that to a list would be fine though.

Reply:
The issue there is I've seen many people not realize that this is something they can change, so I figured making it a list that says TODO makes that clear. I hadn't realized there was a line above so I merged them now